### ISSUE TITLE
Create a `mamba list` for micromamba

### ISSUE THREAD

**Issue by marimeireles** [TheSnakePit/mamba#445]
As discussed with @wolfv, if anyone has anything to add here, please feel free.

**Comment by marimeireles** [comment]
If we `conda list` this happens:
```
# packages in environment at /home/mariana/miniconda3:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       1_gnu    conda-forge
_r-mutex                  1.0.1               anacondar_1    conda-forge
binutils                  2.33.1               h53a641e_8    conda-forge
binutils_impl_linux-64    2.33.1               he1b5a44_7    conda-forge
binutils_linux-64         2.33.1              h9595d00_17    conda-forge
brotlipy                  0.7.0           py37h8f50634_1000    conda-forge
bwidget                   1.9.14                        0    conda-forge
bzip2                     1.0.8                h516909a_2    conda-forge
c-compiler                1.1.1                h516909a_0    conda-forge
ca-certificates           2020.6.20            hecda079_0    conda-forge
caffeine-ng               3.4.2                    pypi_0    pypi
cairo                     1.16.0            h3fc0475_1005    conda-forge
certifi                   2020.6.20        py37hc8dfbb8_0    conda-forge
cffi                      1.14.1           py37he30daa8_0  
chardet                   3.0.4           py37hc8dfbb8_1006    conda-forge
```
I thought a way to prevent this would be using `tabulate`. Any other idea where we don't have to depend on smth else? @SylvainCorlay @JohanMabille

**Comment by marimeireles** [comment]
I think this is how conda people are doing it:

https://github.com/conda/conda/blob/ad15eb51670e6f8213546c67b0c0d2d35db17532/conda/cli/main_list.py#L21-L66

**Comment by marimeireles** [comment]
@wolfv opinions?

**Comment by SylvainCorlay** [comment]
tabulate is probably better, unless crazy folks parse that list...

**Comment by marimeireles** [comment]
hum, just found a bunch of code on mamba
dealing with table printing
https://github.com/TheSnakePit/mamba/blob/96762946c2cacbcfe6f571b4593a222b72430471/src/output.cpp#L277-L383
should prbb use those instead of tabulate, right?
@SylvainCorlay

**Comment by marimeireles** [comment]
Also @wolfv shouldn't the `channel` contain only the channel and not the url? I'm gonna parse the channel out of it for this output. We can move the code to `package_info.cpp` if you think it makes sense.

**Comment by wolfv** [comment]
Yeah we have some code somewhere to cut out the channel url for certain repositories. New code should probably use the name function on the channel class

**Comment by wolfv** [comment]
@JohanMabille  should know about this :)

**Comment by marimeireles** [comment]
@JohanMabille can you point me to this code? I wrote my own function to parse the `channel`.

**Comment by JohanMabille** [comment]
Everything is in [channel.hpp](https://github.com/TheSnakePit/mamba/blob/master/include/mamba/channel.hpp).

I guess the simplest solution is to call `make_channel` and then extract the part that you need:

```cpp
Channel& coco = make_channel(url);
const std::string& name = coco.name();
```

**Comment by marimeireles** [comment]
closed by #470
